src/app/electra/types.ts

```diff
@@ -0,0 +1,69 @@
+
+export interface ElectraState {
```

Comment:
Hm... this doesn't look like a state to me. A state should only contain pure values.

Reply:
So I added this to simulate your electraJs Object